src/test/java/kr/co/wordle/domain/RoundTest.java

```diff
@@ -0,0 +1,47 @@
+package kr.co.wordle.domain;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class RoundTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"abcdef", "asdf", "a", "133ke", "아ddsz", ""})
+    void 입력값_유효하지_않음(String input) {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new Round(input));
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"cigar", "rebut", "sissy", "humph"})
+    void 입력값_유효(String input) {
+        Assertions.assertDoesNotThrow(() -> new Round(input));
+    }
+
+    @Test
+    void 라운드_종료_여부_확인() {
+        String input = "cigar";
+        Round round = new Round(input);
+        Assertions.assertFalse(round.isFinished());
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"cigar:cigar:GREEN", "stink:marry:GRAY", "marry:karma:YELLOW"}, delimiter = ':')
+    void 정답과_비교(String input, String strAnswer, String tile) {
+        Round round = new Round(input);
+        Answer answer = new Answer(strAnswer);
+        Tile expected = Tile.valueOf(tile);
+        Tile result = round.getTile(answer.getCountPerCharacter(), answer.charAt(0), input.charAt(0));
+        Assertions.assertEquals(expected, result);
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"cigar:cigar:🟩🟩🟩🟩🟩", "marry:karma:🟨🟩🟩⬜️⬜️", "major:marry:🟩🟩⬜️⬜️🟨"}, delimiter = ':')
```

Comment:
저랑 딱구님이 정답 테스트 할 때 사용한 단어로 테스트 해봤어요!
- answer : apple
- input : hello
```suggestion
    @CsvSource(value = {"cigar:cigar:🟩🟩🟩🟩🟩", "marry:karma:🟨🟩🟩⬜️⬜️", "major:marry:🟩🟩⬜️⬜️🟨", "hello:apple:⬜️🟨⬜️🟩⬜️"}, delimiter = ':')
```
Expected :⬜️🟨⬜️🟩⬜️
Actual   :⬜️🟨🟨🟩⬜️

추가한 테스트 케이스에 대해 실패하는 것을 확인했는데요, 
입력한 알파벳 중 동일한 알파벳이 여러 개일 경우 그 중 동일한 위치만 초록색이고 나머지는 흰색이 되어야 하는 케이스를 추가해서 알고리즘을 수정해보면 좋을 것 같아요! (첫 번째 `l`은 포함되지만 이미 두 번째 `l`이 초록색이기 때문에 흰색)

Reply:
수정 완료입니다 ! 짚어주셔서 감사해요 ☺️